atomic_reactor/plugins/post_koji_upload.py

```diff
@@ -62,9 +59,9 @@ def callback(self, offset, totalsize, size, t1, t2): # pylint: disable=W0613
                               percent_done, size / t1 / 1024 / 1024)
 
 
-class KojiPromotePlugin(ExitPlugin):
+class KojiUploadPlugin(PostBuildPlugin):
     """
-    Promote this build to Koji
+    Upload this build to Koji
 
     Submits a successful build to Koji using the Content Generator API,
     https://fedoraproject.org/wiki/Koji/ContentGenerators
```

Comment:
We should remove 'Submits a successful build to Koji using the Content Generator API', because that's what koji-import will do. How about this?:
```
Note: only the image archive is uploaded to Koji 
at this stage. Metadata about this image
is created and stored in a ConfigMap in 
OpenShift, ready for the orchestrator build to
collect and use to actually create the Koji Build 
together with the uploaded image archive(s).
```

We also no longer need the `metadata_only` parameter. It can be removed entirely from koji_upload.